cfg/config_test.go

```diff
@@ -139,6 +144,14 @@ func TestParseExpressionConst(t *testing.T) {
 	assert.Equal(t, 10, s.T_, "wrong value")
 }
 
+func TestParseUnit(t *testing.T) {
+	s := &strUnit{T: "10 MB"}
```

Comment:
Can we add more test cases?
For example, T: "1", T: "1 ", T: ""

Reply:
How about "10MB", "MB", "blablabla", "999999999999999999999" and some other crazy stuff? It may seem crazy for us at first but at the end of the day it's a config file, ppl other than us may write them in the future.  These cases should be covered as well IMO. 
Also, do we need a separate test function for every "wrong" format we test in here? Maybe it's a good idea to combine some cases together?